src/trinary.c

```diff
@@ -133,6 +133,9 @@ Trobject_t *trytes_from_trits(Trobject_t *trits)
         return NULL;
     }
 
+#if defined(__SSE4_2__)
+    return trytes_from_trits_sse42(trits);
+#else
```

Comment:
We can safely replace `#else` with `#endif` since it exists as a shortcut.

Reply:
Done with the additional commit `refactor: Remove redundant conditional compilation`.